lib/hardware/led_light.hpp

```diff
@@ -0,0 +1,17 @@
+#ifndef PET_PASSAGE_HARDWARE_LED_LIGHT_H_
+#define PET_PASSAGE_HARDWARE_LED_LIGHT_H_
+
+#include "Arduino.h"
+#include "led_base.hpp"
+
+template <uint8_t Pin>
+class LedLight : public LedBase {
```

Comment:
Why call this `LedLight`? I think it would be better to call it simply `Led` (adding Light seemes superfluous)

Reply:
Additionally, probably mark this as final (we will need to do this for other classes later)